### PR TITLE
Docs(web-twig): Improve additional API options info #DS-1099

### PR DESCRIPTION
**Pull request by crishpeen**
Awaiting your review!

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Header/README.md [hunk 1]
```diff
@@ -112,10 +112,9 @@ just branding.
 | `color`    | [`transparent` \| `inverted`] | `transparent` | ✕        | Color variant                       |
 | `isSimple` | `bool`                        | `false`       | ✕        | Shorter, centered version of Header |
 
-On top of the API options, you can add `data-*` or `aria-*` attributes to
-further extend component's descriptiveness and accessibility. Also, UNSAFE styling props are available,
-see the [Escape hatches][escape-hatches] section in README to learn how and when to use them.
-These attributes will be passed to the topmost HTML element of the component.
+On top of the API options, the components accept [additional attributes][readme-additional-attributes].
+If you need more control over the styling of a component, you can use [styling props][readme-styling-props]
+and [escape hatches][readme-escape-hatches].
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Header/README.md:117]: Maybe "styling escape hatches".

**Review by pavelklibani — APPROVED** [review]

**Comment by literat** [comment]
I have added one comment about escape hatches. Otherwise, it is good to go for me. Good Job 👍

File: packages/web-twig/src/Resources/components/Header/README.md (hunk 1 above)
Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Header/README.md:117]: Shouldn't we change the title in the docs too then?

File: packages/web-twig/README.md [hunk 2]
```diff
@@ -78,6 +78,21 @@ For available components see the [components directory](https://github.com/lmc-e
...
 
+## Additional Attributes
+
+All components accept additional attributes that are passed down to the root element of the component.
+This is useful for adding custom event handlers, accessibility attributes, or other attributes that
+are not supported by the component API.
+
+Supported attributes are:
+
+- `on*` (eg. `onclick`)
```
Comment (adamkudrna) [inline packages/web-twig/README.md:89]: Please explain how the `on*` attributes/props are passed to the native element (`<input>`, `<button>`, …).

File: packages/web-twig/src/Resources/components/Header/README.md (hunk 1 above)
Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/Header/README.md:117]: It's open for a future introduction of other escape hatches, isn't it?

Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Header/README.md:117]: we agreed on 
> you can use [style props][readme-style-props]
and [escape hatches][readme-escape-hatches].

File: packages/web-twig/src/Resources/components/Button/README.md [hunk 3]
```diff
@@ -46,11 +46,13 @@ Without lexer:
 | `size`       | [Size dictionary][dictionary-size]                                                        | `medium`  | ✕        | Size variant                                                               |
 | `type`       | `string`                                                                                  | `button`  | ✕        | Type of the Button                                                         |
 
-You can add `id`, `data-*` or `aria-*` attributes to further extend component's
-descriptiveness and accessibility. Also, UNSAFE styling props are available,
-see the [Escape hatches][escape-hatches] section in README to learn how and when to use them.
+YOn top of the API options, the components accept [additional attributes][readme-additional-attributes].
```
Comment (pavelklibani) [inline packages/web-twig/src/Resources/components/Button/README.md:49]:
```suggestion
On top of the API options, the components accept [additional attributes][readme-additional-attributes].
```

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**